packages/client/components/ActivityLibrary/ActivityLibraryCard.tsx

```diff
@@ -8,7 +8,7 @@ export const ActivityLibraryCard = (props: ActivityCardProps) => {
   return (
     <ActivityCard
       className={clsx(
-        'group/card transition-shadow focus-within:ring-2 focus-within:ring-primary hover:shadow-md motion-reduce:transition-none',
+        'transition-shadow focus-within:ring-2 focus-within:ring-primary hover:shadow-md',
```

Comment:
+1 I personally am not a big fan of the `transition-shadow` as it feels less snappy. If it would be a different thing that we'd animate, it might be different.